community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/README.md

```diff
@@ -0,0 +1,204 @@
+## Description
+This module is a wrapper around the [slurm-controller-hybrid] module by SchedMD
+as part of the [slurm-gcp] github repository. The hybrid module serves to create
+the configurations needed to extend an on-premise slurm cluster to one with one
+or more Google Cloud bursting partitions. These partitions will create the
+requested nodes in a GCP project on-demand and scale after a period of not being
+used, in the same way as the [schedmd-slurm-gcp-v5-controller] module
+auto-scales VMs.
+
+> **_NOTE:_** This module has only been tested in limited capacity with the HPC
+> Toolkit. On Premise configurations can vary significantly, this module should
+> be used as a starting point, not a complete solution.
+
+[schedmd-slurm-gcp-v5-controller]: ../schedmd-slurm-gcp-v5-controller/
+
+### Usage
+The [slurm-controller-hybrid] is intended to be run on the controller of the on
+premise slurm cluster, meaning executing `terraform init/apply` against the
+deployment directory. This allows the module to infer settings such as the
+slurm user and user ID when setting permissions for the created configurations.
+
+If unable to install terraform and other dependencies on the controller
+directly, it is possible to deploy the hybrid module in a separate build
+environment and copy the created configurations to the on premise controller
+manually. This will require addition configuration and verification of
+permissions. For more information see the [hybrid.md] documentation on
+[slurm-gcp].
+
+[slurm-controller-hybrid]: https://github.com/SchedMD/slurm-gcp/tree/v5.0.3/terraform/slurm_cluster/modules/slurm_controller_hybrid
+
+> **_NOTE:_** The hybrid module requires the following dependencies to be
+> installed on the system deploying the module:
+>
+> * [terraform]
+> * [addict]
+> * [google-cloud-pubsub]
+> * A full list of recommended python packages is available in a
+>   [requirements.txt] file in the [slurm-gcp] repo.
```

Comment:
When testing this in a clean virtual environment I needed to install the following:
```
pip install addict httplib2 google-api-python-client pyyaml
```
I have confirmed that httplib2 and pyyaml are not dependencies of `google-cloud-pubsub`, I would recommend updating the list to what I observed.

Reply:
Added

Reply:
nit: it is not clear that terraform is the terraform binary and the rest are python pip packages. We can clean up in our next pass.